teflo_datarouter_plugin/datarouter_plugin.py

```diff
@@ -138,6 +123,14 @@ def send_put_req(self, access_token, dr_url, tar_payload, json_config_file):
         except Exception as ex:
             raise TefloReportError(f'Failed Sending PUT req with error: {ex}')
 
+    def get_req_status(self, req_id, ac_token, dr_token_url):
+
+        """GET REQ STATUS"""
+        req_url = self.provider_credentials['host_url'] + '/api/requests/'
+        get_url = req_url+req_id
+        get_call = get_req_status(get_url, ac_token, self.dr_client_id, self.dr_client_secret, dr_token_url)
```

Comment:
modify this logic get_req_status is being called within itself. You dont need to have this methid, just call the one from the helper class